react-components/src/components/CacheProvider/FdmNodeCache.ts

```diff
@@ -339,20 +337,20 @@ async function createRevisionToConnectionsMap(
 
 function createFdmConnectionWithNode(
   modelRevisionId: ModelRevisionId,
-  modelNodeIdToNodeMap: Map<ModelNodeIdKey, Node3D>,
+  modelTreeIndexToNodeMap: Map<ModelTreeIndexKey, Node3D>,
   connection: FdmCadConnection,
-  view?: Source
+  views?: Source[]
```

Comment:
Is there any reason to pass the views here even not really used inside the function? Couldnt be just integrated into the object outside where is called?

Reply:
I do see your point.. But I actually kind-of like that the value is created immutably here, and the connection isn't added later. I think I want to keep it :see_no_evil: